uint64 distance
uint64 search_length
uint32 sweep_width

geometry_msgs/Point[] waypoints
